### ISSUE TITLE
Module build failed: BrowserslistError: Unknown browser query `android all`

### ISSUE THREAD

**Issue by wanglijie** [ben-eb/caniuse-lite#27]
Upon your last update version on npm，module build causes postcss-preset-env throwing error.

**Comment by wanglijie** [comment]
![image](https://user-images.githubusercontent.com/3660583/59666680-e65f7800-91e7-11e9-8480-287559da13a3.png)

**Comment by wojtii** [comment]
Related threads:
https://github.com/facebook/create-react-app/issues/7239
https://github.com/browserslist/browserslist/issues/382

**Comment by timoken** [comment]
People at that [create-react-app issue](https://github.com/facebook/create-react-app/issues/7239) are like

![simp](https://user-images.githubusercontent.com/38980986/59669634-dd0be680-91ba-11e9-8709-6dc85b0cf709.jpg)

🤣

**Comment by robinsingh-bw** [comment]
For good reason...

**Comment by cathcheeno** [comment]
Plz..

**Comment by ai** [comment]
We made a dirty fix in Browserslist 4.6.3 to fix the problem:

https://twitter.com/Browserslist/status/1140911400304467968

**Comment by wanglijie** [comment]
> We made a dirty fix in Browserslist 4.6.3 to fix the problem:
> 
> https://twitter.com/Browserslist/status/1140911400304467968

It works, tons of BrowserslistError warning showing up though.

![image](https://user-images.githubusercontent.com/3660583/59670425-828c7d80-91ee-11e9-8d73-f09a74ecaabc.png)

**Comment by 974N** [comment]
I tried to run npm i postcss-preset-env through terminal, ran the react app but still same problem.
```
Failed to compile
./src/index.css (./node_modules/css-loader/dist/cjs.js??ref--6-oneOf-3-1!./node_modules/postcss-loader/src??postcss!./src/index.css)
BrowserslistError: Unknown browser query `android all`. Maybe you are using old Browserslist or made typo in query.
    at Array.reduce (<anonymous>)
    at Array.some (<anonymous>)
    at Array.filter (<anonymous>)
```

**Comment by wanglijie** [comment]
> I tried to run npm i postcss-preset-env through terminal, ran the react app but still same problem.
> 
> ```
> Failed to compile
> ./src/index.css (./node_modules/css-loader/dist/cjs.js??ref--6-oneOf-3-1!./node_modules/postcss-loader/src??postcss!./src/index.css)
> BrowserslistError: Unknown browser query `android all`. Maybe you are using old Browserslist or made typo in query.
>     at Array.reduce (<anonymous>)
>     at Array.some (<anonymous>)
>     at Array.filter (<anonymous>)
> ```

Try installing latest version of postcss-preset-env

**Comment by jrose-carecloud** [comment]
Having the same issue. This is a major problem for us right now.

**Comment by niftylettuce** [comment]
@ai can you please `npm unpublish caniuse-browserlist@1.0.30000975`?

**Comment by grzegorzstachniukalm** [comment]
for me temporary solution was just 
```json
{
  "resolutions": {
    "graphql": "14.3.1",
    "@types\/react": "16.8.20",
    "caniuse-lite": "1.0.30000971"
  }
}
```

**Comment by mresmaeili** [comment]
this just made me forget the whole problem

**Comment by DrunkenPoney** [comment]
Maybe a rollback to the previous version would be a good idea?
At least until the problem is fixed.

**Comment by ai** [comment]
@DrunkenPoney we already have a dirty fix in Browserslist and now working on a proper fix in `postcss-preset-env`.

Anyway `caniuse-lite` is just a repack of `caniuse-db`. The error was in `caniuse-db`, but Can I Use authors doesn’t want to do anything with it and give it as-is.

**Comment by ehdwn1212** [comment]
Same issue

**Comment by testcderick** [comment]
Builds is okay now, since the latest update to caniuse-lite from 5 hours ago.

Update the version of caniuse 1.0.30000976 and everything will be fine.

**Comment by adi518** [comment]
> > We made a dirty fix in Browserslist 4.6.3 to fix the problem:
> > https://twitter.com/Browserslist/status/1140911400304467968
> 
> It works, tons of BrowserslistError warning showing up though.
> 
> ![image](https://user-images.githubusercontent.com/3660583/59670425-828c7d80-91ee-11e9-8d73-f09a74ecaabc.png)

How did it work if you still get those console logs?

**Comment by ai** [comment]
The problem was fixed
